Report/Chapters/Implementation.tex

```diff
@@ -3,6 +3,20 @@ \chapter{Implementation}
 
 \section{Front end}
 
+\subsection{ContextMenu}
+
+In order to enable context specific actions in a touch environment we needed some kind of specific menu.
+The CircleMenu class, an extension of the CirclePopupMenu of the JFXtras library, fulfils this role by spawning a series of icons that trigger context specific actions such as: copy, paste, save and delete.
```

Comment:
'fulfills'. 'Spawning' is volgens mij een game-term, 'showing' is misschien logischer hier. De dubbele punt is misschien niet nodig.